SUPPORTED_SCENARIOS.md

```diff
@@ -58,3 +58,7 @@ The goal of this document is to have a single source of information  which scena
 * /boot
 * extended partition
   * /boot has to be on a logical partition
+  
+# Required software
+
+* During installation, yast-bootloader requires installing the packages kexec_tools and grub2. If one decides for not installing grub in MBR <b>and</b> "/" then the system can be installed without these packages.
```

Comment:
see e.g. grub2-efi packages, which require also shim if secure boot is used https://github.com/yast/yast-bootloader/blob/master/src/lib/bootloader/grub2efi.rb#L94

and on other hand kexec-tools is not always installed see https://github.com/yast/yast-bootloader/blob/master/src/lib/bootloader/bootloader_base.rb#L56